app/results.py

```diff
@@ -188,8 +188,12 @@ def get(self):
             # If a query looks like a phone number, show the user a result
             # of looking up the number in the carriers-provided BBS system.
             if self.config.jp_mobile_carrier_redirect:
-                if jp_mobile_carriers.handle_phone_number(self, query.query):
-                    return 
+                try:
+                    if jp_mobile_carriers.handle_phone_number(self, query.query):
+                        return
+                except Exception as e:
+                    logging.error('failed to scrape phone number')
```

Comment:
Can you also show the error message to the user too? You can use self.error like this:
https://github.com/google/personfinder/blob/master/app/view.py#L175

Also, "scrape phone number" is confusing because it sounds like we collect phone numbers from some page. You should say e.g., "scrape the search result for the phone number".